modules/editable-layers/src/edit-modes/extrude-mode.ts

```diff
@@ -61,6 +62,10 @@ export class ExtrudeMode extends ModifyMode {
   }
 
   handleStartDragging(event: StartDraggingEvent, props: ModeProps<FeatureCollection>) {
+    if (shouldCancelPan(event)) {
```

Comment:
Does this need to be implemented by every mode? Could it be called by the framework before invoking `handleStartDragging`? 

Do we need a `mode writer's guide` page in the docs that explains this?